e2e/shared/common.js

```diff
@@ -248,10 +255,35 @@ function removeAppFolder(folderPath) {
   const resolvedFolderPath = path.join(path.resolve(tmp), 'src', 'app', folderPath);
   return new Promise((resolve, reject) => {
     if (fs.existsSync(resolvedFolderPath)) {
-      fs.rmdirSync(resolvedFolderPath);
+      /**
```

Comment:
Have you thought about importing `fs-extra` and calling `emptyDirSync` instead?

Reply:
I did not know that was a thing.   But I suppose that would be less code.

Reply:
Good call on using `fs-extra` saved a ton of code and dried up some of the methods we had going.